src/utils.js

```diff
@@ -0,0 +1,48 @@
+const spawn = require("cross-spawn");
+const path = require("path");
+
+module.exports.runCordova = (args, envArgs = {}) => {
+  const cordovaCli = path.join(
+    process.cwd(),
+    "node_modules",
+    ".bin",
+    "cordova"
+  );
+  const cordovaDir = path.join(process.cwd(), "cordova");
+
+  return runCmd(cordovaCli, args, envArgs, { cwd: cordovaDir });
+};
+
+module.exports.runYarn = (args, envArgs = {}) => {
+  return runCmd("yarn", ["--silent", ...Array.from(args)], envArgs);
+};
+
+const runCmd = (cmd, args, envArgs = {}, spawnArgs = {}) => {
+  if (process.env.TRACE) {
+    console.log(
+      `Executing ${cmd} (${args}) (${JSON.stringify(
+        envArgs
+      )}) (${JSON.stringify(spawnArgs)})`
+    );
+  }
+
+  return new Promise((resolve, reject) => {
+    let env = Object.create(process.env);
+    Object.assign(env, envArgs);
+
+    let child = spawn(cmd, args, { env, stdio: "inherit", ...spawnArgs });
+
+    if (child.stdout != null) {
+      child.stdout.on("data", data => process.stdout.write(data));
```

Comment:
I thought that `stdio: "inherit"` already pipes the child's `stdout` and `stderr` to the parent, but I could be wrong.

Reply:
Could well be, I just moved this code over from the previous implementation. Actually we might as well use spawnSync with `stdio: inherit` as async is not important here anyway.